lib/ReactViews/Workbench/WorkbenchItem.tsx

```diff
@@ -22,187 +21,167 @@ import { TextSpan } from "../../Styled/Text";
    */
-  @computed
-  get isOpen(): boolean {
-    if (!CatalogMemberMixin.isMixedInto(this.props.item)) return true;
-    return this.props.item.isOpenInWorkbench;
-  }
+  const isOpen =
+    !CatalogMemberMixin.isMixedInto(item) || item.isOpenInWorkbench;
 
-  render() {
-    const { item, t } = this.props;
+  const isLoading =
+    (CatalogMemberMixin.isMixedInto(item) && item.isLoading) ||
+    (ReferenceMixin.isMixedInto(item) && item.isLoadingReference);
 
-    const isLoading =
-      (CatalogMemberMixin.isMixedInto(item) && item.isLoading) ||
-      (ReferenceMixin.isMixedInto(item) && item.isLoadingReference);
-
-    return (
-      <StyledLi style={this.props.style} className={this.props.className}>
-        <Box
-          fullWidth
-          justifySpaceBetween
-          paddedRatio={3}
-          styledMinHeight="38px"
-        >
-          <Box fullWidth>
-            <Box left fullWidth centered>
-              <DraggableBox
-                onMouseDown={this.props.onMouseDown}
-                onTouchStart={this.props.onTouchStart}
-                title={getPath(item, " → ")}
-                fullWidth
-              >
-                {!(item as any).isMappable && !isLoading && (
-                  <BoxSpan paddedHorizontally displayInlineBlock>
-                    <Box padded>
-                      <StyledIcon
-                        styledHeight={"18px"}
-                        light
-                        glyph={Icon.GLYPHS.lineChart}
-                      />
-                    </Box>
-                  </BoxSpan>
-                )}
-                {MappableMixin.isMixedInto(item) ? (
-                  <Box left verticalCenter>
-                    <Checkbox
-                      id="workbenchtoggleVisibility"
-                      isChecked={item.show}
-                      isSwitch
-                      title={t("workbench.toggleVisibility")}
-                      onChange={() => this.toggleVisibility()}
-                      css={`
-                        overflow-wrap: anywhere;
-                        margin-right: 5px;
-                      `}
-                      textProps={{ medium: true, fullWidth: true }}
-                    >
-                      <TextSpan
-                        medium
-                        maxLines={!this.isOpen ? 2 : false}
-                        title={getName(item)}
-                      >
-                        {getName(item)}
-                      </TextSpan>
-                    </Checkbox>
+  return (
+    <StyledLi style={style} className={className}>
+      <Box fullWidth justifySpaceBetween paddedRatio={3} styledMinHeight="38px">
+        <Box fullWidth>
+          <Box left fullWidth centered>
+            <DraggableBox
+              onMouseDown={onMouseDown}
+              onTouchStart={onTouchStart}
+              title={getPath(item, " → ")}
+              fullWidth
+            >
+              {!(item as any).isMappable && !isLoading && (
```

Comment:
`MappableMixin.isMixedInto(item) && !item.isMappable && !isLoading && (`?

Reply:
This is for the case when `isMappable` is false and `MappableMixin.isMixedInto` check `model.isMappable`